internal/sqlparser/parser.go

```diff
@@ -11,6 +11,20 @@ import (
 	"sync"
 )
 
+type Direction string
```

Comment:
The `ParseSQLMigration` took 2 booleans with a signature like `(r io.Reader, direction, verbose bool)`.

It's waaaay too easy for the caller to mix up the booleans, so instead, the second param is now a well-defined type.